<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GoToShutdown</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>icons/Alecive-Flatwoken-Apps-Dialog-Shutdown.ico</normalon>
   </iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background : qlineargradient(spread:pad, x1:0.417, y1:0.3125, x2:1, y2:0, stop:0 rgba(0, 10, 76, 255), stop:0.810945 rgba(186, 97, 220, 255));</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <property name="dockNestingEnabled">
   <bool>true</bool>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowNestedDocks|QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QToolButton" name="settingsBtn">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>10</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background : transparent;
color : white;
border : 1px solid white;
border-radius : 5px;</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startBtn">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>520</y>
      <width>93</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background : transparent;
color : white;
border : 1px solid white;
border-radius : 5px;</string>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addBtn">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>520</y>
      <width>93</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background : transparent;
color : white;
border : 1px solid white;
border-radius : 5px;</string>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="abortBtn">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>520</y>
      <width>93</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background : transparent;
color : white;
border : 1px solid white;
border-radius : 5px;</string>
    </property>
    <property name="text">
     <string>Abort</string>
    </property>
   </widget>
   <widget class="QTimeEdit" name="timeEdit">
    <property name="geometry">
     <rect>
      <x>347</x>
      <y>270</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background : transparent;
color : white;
border : 1px solid white;
border-radius : 5px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="logo">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>20</y>
      <width>141</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background : transparent;
color : white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
    <property name="pixmap">
     <pixmap>icons/Alecive-Flatwoken-Apps-Dialog-Shutdown.ico</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QScrollArea" name="timesList">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>141</width>
      <height>231</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">background : transparent;
color : white;
border : 1px solid white;
border-radius : 5px;</string>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>139</width>
       <height>229</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2"/>
    </widget>
   </widget>
   <widget class="QPushButton" name="deleteBtn">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>280</y>
      <width>41</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background : transparent;
color : white;
border-radius : 5px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normalon>icons/remove.png</normalon>
     </iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="editBtn">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>280</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">background : transparent;
color : white;
border-radius : 5px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normalon>icons/pencil.png</normalon>
     </iconset>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
